alembic/autogenerate/render.py

```diff
@@ -457,40 +457,56 @@ def _drop_constraint(
 
 @renderers.dispatch_for(ops.AddColumnOp)
 def _add_column(autogen_context: AutogenContext, op: ops.AddColumnOp) -> str:
-    schema, tname, column = op.schema, op.table_name, op.column
+    schema, tname, column, postgresql_if_not_exists = (
+        op.schema,
+        op.table_name,
+        op.column,
+        op.postgresql_if_not_exists,
```

Comment:
we can't have postgresql-specific logic in render here. but also, the op should not have a `.postgresql_if_not_exists` field anyway, it should be in `.kw`.    

while all of the ops have a `.kw` field that stores additional dialect-specific keyword arguments, these are not included in renders right now.    we can try adding these to renders but this would need tests across many 

that said it's not clear if this argument should be called `postgresql_if_not_exists` anyway